src/main/charts/bitbucket/templates/statefulset.yaml

```diff
@@ -76,15 +76,17 @@ spec:
             {{ if .Values.ingress.host }}
             - name: SERVER_PROXY_NAME
               value: {{ .Values.ingress.host | quote }}
+            - name: SERVER_PROXY_PORT
+              value: {{ include "bitbucket.ingressPort" . | quote }}
             - name: SETUP_BASEURL
               value: {{ include "bitbucket.baseUrl" . | quote }}
             {{ end }}
-            - name: SERVER_PROXY_PORT
-              value: {{ .Values.ingress.port | quote }}
+            {{ if .Values.ingress.https }}
             - name: SERVER_SCHEME
-              value: {{ .Values.ingress.scheme | quote }}
+              value: "https"
             - name: SERVER_SECURE
-              value: {{ .Values.ingress.secure | quote }}
+              value: "true"
+            {{ end }}
```

Comment:
Now we set the `SERVER_SCHEME` and `SERVER_SECURE` env vars on the basis of a singe chart value, rather than one chart value for each.